Shared/Features/Projects/View/ProjectsView.swift

```diff
@@ -1,8 +1,28 @@
 import SwiftUI
 
 struct ProjectsView: View {
+    init() {
+        let coloredNavAppearance = UINavigationBarAppearance()
+        coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor(Color("SecondaryColor"))]
+        coloredNavAppearance.backgroundColor = UIColor(Color("AccentColor"))
+
+        UINavigationBar.appearance().standardAppearance = coloredNavAppearance
+        UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
+    }
+
     var body: some View {
-        Text("Projects")
+        NavigationView {
+            Text("Projects")
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .principal) {
+                        Text("Projects")
+                            .fontWeight(.medium)
+                            .font(.system(size: 24))
```

Comment:
Here is a great opportunity to learn more about SwiftUI modifiers and to apply them: https://www.hackingwithswift.com/books/ios-swiftui/custom-modifiers